stdlib/public/Darwin/Accelerate/Quadrature.swift

```diff
@@ -0,0 +1,258 @@
+            case .qag(let pointsPerInterval, let maxIntervals):
+                integrateOptions.integrator = QUADRATURE_INTEGRATE_QAG
+                integrateOptions.qag_points_per_interval = pointsPerInterval.rawValue
+                integrateOptions.max_intervals = maxIntervals
+            case .qags(let maxIntervals):
+                integrateOptions.integrator = QUADRATURE_INTEGRATE_QAGS
+                integrateOptions.max_intervals = maxIntervals
+            }
+        }
+        get {
+            let integrator: Integrator!
+            
+            switch integrateOptions.integrator {
+            case QUADRATURE_INTEGRATE_QNG:
+                integrator = .qng
+            case QUADRATURE_INTEGRATE_QAG:
+                integrator = .qag(pointsPerInterval: QAGPointsPerInterval(rawValue: integrateOptions.qag_points_per_interval)!,
+                                  maxIntervals: integrateOptions.max_intervals)
+            case QUADRATURE_INTEGRATE_QAGS:
+                integrator = .qags(maxIntervals: integrateOptions.max_intervals)
+            default:
+                integrator = nil
+            }
+            
+            return integrator
+        }
+    }
+    
+    /// The quadrature instance's requested absolute tolerance on the result.
+    public var absoluteTolerance: Double {
+        set {
+            integrateOptions.abs_tolerance = newValue
+        }
+        get {
+            return integrateOptions.abs_tolerance
+        }
+    }
+    
+    /// The quadrature instance's requested relative tolerance on the result.
+    public var relativeTolerance: Double {
+        set {
+            integrateOptions.rel_tolerance = newValue
+        }
+        get {
+            return integrateOptions.rel_tolerance
+        }
+    }
+    
+    /// The maximum number of subintervals in the subdivision used by QAG and QAGS integrators.
+    public var maxIntervals: Int {
+        set {
+            integrateOptions.max_intervals = newValue
+        }
+        get {
+            return integrateOptions.max_intervals
+        }
+    }
+    
+    /// Number of points per subinterval. Used by the QAG integrator only; other integrators ignore this value.
+    public var qagPointsPerInterval: QAGPointsPerInterval {
+        set {
+            integrateOptions.qag_points_per_interval = newValue.rawValue
+        }
+        get {
+            return QAGPointsPerInterval(rawValue: integrateOptions.qag_points_per_interval) ?? .default
+        }
+    }
+    
+    /// Performs the integration over the supplied function.
+    ///
+    /// - Parameter interval: The lower and upper bounds of the integration interval.
+    /// - Parameter integrand: The function to integrate. The input value is `x` that's within the interval over which the integrand is being integrated, and the output value is the corresponding value `y = integrand(x)` at those points.
+    public func integrate(over interval: ClosedRange<Double>,
```

Comment:
There should also be a version of this func that exposes the vectorized callback, since you're adding all these nice vectorized vForce functions. Specifically, it would be great to expose something like:

`public func integrate(over interval: ClosedRange<Double>, integrand: (UnsafeBufferPointer<Double>, inout UnsafeMutableBufferPointer<Double>) -> ())`

so that `integrand` can be implemented in terms of the vForce and vDSP interfaces.